tensorflow/python/ops/resource_variable_ops.py

```diff
@@ -1759,7 +1759,7 @@ def _init_from_args(self,
         initial_value, "graph") and initial_value.graph.building_function:
       raise ValueError(f"Argument `initial_value` ({initial_value}) could not "
                        "be lifted out of a `tf.function`. "
-                       "(Tried to create variable with name='{name}'). "
+                       f"(Tried to create variable with name='{name}'). "
```

Comment:
again, this is a spurious change - perhaps check your checker script so that these multi-line literals are ignored if they already have the leading f?